opaque_keys/__init__.py

```diff
@@ -357,6 +402,8 @@ def __setstate__(self, state_dict):
             if key in self.KEY_FIELDS:  # pylint: disable=no-member
                 setattr(self, key, state_dict[key])
         self.deprecated = state_dict['deprecated']  # pylint: disable=assigning-non-slot
+        self._unicode = None  # pylint: disable=assigning-non-slot
```

Comment:
Yeah, getting rid of that causes pylint to say: `E:309, 8: Assigning to attribute '_unicode' not defined in class slots (assigning-non-slot)`. :-/